rust/osm2lanes/src/locale.rs

```diff
@@ -171,6 +171,15 @@ impl std::str::FromStr for DrivingSide {
     }
 }
 
+impl ToString for DrivingSide {
+    fn to_string(&self) -> String {
+        match self {
+            Self::Right => String::from("right"),
```

Comment:
not sure if important,

but I avoided implementing this to avoid confusing right hand drive with right hand travel.